node/sc/local_backend.go

```diff
@@ -152,11 +157,11 @@ func (lb *LocalBackend) PendingNonceAt(ctx context.Context, account common.Addre
 	return lb.subbrige.txPool.GetPendingNonce(account), nil
 }
 
-func (lb *LocalBackend) SuggestGasPrice(ctx context.Context) (*big.Int, error) {
+func (lb *LocalBackend) SuggestGasPrice(_dummyCtx context.Context) (*big.Int, error) {
```

Comment:
Why did you change `ctx` to `_dummyCtx` only in a few methods? It seems that other methods such as `PendingNonceAt` doesn't use `ctx` but you didn't change the name of it.

Reply:
_dummyCtx name is not good if it remains in codes, because other readers are confused, they don't know why it is used. When you want to rename unused parameters, it is good to remove as possible.